src/NexusMods.Common/ApplicationConstants.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Diagnostics;
+
+namespace NexusMods.Common;
+
+/// <summary>
+/// Constants supplied during runtime.
+/// </summary>
+public static class ApplicationConstants
+{
+    /// <summary>
+    /// The current version of the app.
+    /// </summary>
+    public static Version CurrentVersion
+    {
+        get
+        {
+
+            if (CompileConstants.InstallationMethod == InstallationMethod.Manually || CompileConstants.IsDebug)
+                return Version.Parse("0.0.0.1");
```

Comment:
What's this magic number?

Reply:
Low enough to trigger a update no matter what version we are on.